core/tests/test_utils.py

```diff
@@ -1407,6 +1407,12 @@ def save_new_story(
         Returns:
             Story. A newly-created story.
         """
+        if corresponding_topic_id is None:
```

Comment:
Don't do this. Force corresponding_topic_id to be passed in and put it before the optional args. The behaviour is otherwise too indirect.